website/docs/docs/integrate/dbt-semantic-layer.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "dbt Semantic Layer"
+id: dbt-semantic-layer
+description: "Introducing the dbt Semantic Layer"
+sidebar_label: "dbt Semantic Layer"
+---
+
+:::info 📌
+
+The dbt Semantic Layer is currently available for public preview! With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
+
+For more info on public preview and plan availability, check out the [public preview](/docs/integrate/dbt-semantic-layer#public-preview) and [product architecture](/docs/integrate/dbt-semantic-layer#product-architecture) section.
+    
+:::
+
+The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project), for consistent self-service within downstream data tools like BI and metadata management solutions. 
+
+The result? You have less duplicative coding for data teams and more consistency for data consumers. The dbt Semantic Layer has four main parts:
+
+- Define your metrics in version-controlled dbt project code
+- Import your metric definitions via the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying)
+- Query your metric data via the dbt Proxy Server
+- Explore and analyze dbt metrics in downstream tools
+
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl_architecture.png" title="dbt Semantic Layer architecture" />
+
+**What makes the dbt Semantic Layer different?**
+
+The dbt Semantic Layer removes code duplication and inconsistency when it comes to your business metrics. By moving metric definitions out of the BI layer and into the modeling layer, data teams can feel confident that different business units are working from the same data definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invoked and creates consistency across all applications.
+
+
+## Prerequisites
+To use the dbt Semantic Layer, you’ll need to meet the following:
+
+- Have a multi-tenant [dbt Cloud](https://cloud.getdbt.com/) Teams or Enterprise account. 
+   * Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API.
+- Have both your production and development environments running dbt version 1.2 (latest) or higher
+- Use Snowflake data platfrom 
+- Install the metrics package version 0.3.2 or higher in your dbt project
+- Set up the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview) in the integrated tool to import metric definitions
+- Recommended - Review the dbt metrics page and Getting started with the dbt Semantic Layer blog
```

Comment:
It might be beneficial to link to these pages again so readers don't have to scroll and find.